requirements.txt

```diff
@@ -0,0 +1,38 @@
+annotated-types==0.7.0
+anyio==4.9.0
+cachetools==5.5.2
+certifi==2025.7.9
+charset-normalizer==3.4.2
+click==8.2.1
+fastapi==0.116.0
+google-ai-generativelanguage==0.6.15
+google-api-core==2.25.1
+google-api-python-client==2.176.0
+google-auth==2.40.3
+google-auth-httplib2==0.2.0
+google-generativeai==0.8.5
+googleapis-common-protos==1.70.0
+grpcio==1.73.1
+grpcio-status==1.71.2
+h11==0.16.0
+httplib2==0.22.0
+idna==3.10
+proto-plus==1.26.1
+protobuf==5.29.5
+pyasn1==0.6.1
+pyasn1_modules==0.4.2
+pydantic==2.11.7
+pydantic_core==2.33.2
+pyparsing==3.2.3
+python-dotenv==1.1.1
+requests==2.32.4
+rsa==4.9.1
+sniffio==1.3.1
+starlette==0.46.2
+tqdm==4.67.1
+typing-inspection==0.4.1
+typing_extensions==4.14.1
+uritemplate==4.2.0
+urllib3==2.5.0
+uvicorn==0.35.0
+python-jose[cryptograghy]
```

Comment:
Please update the `requirements.txt` file to fix the typo:
- **Line:** `python-jose[cryptography]`
- **Change:** `cryptograGhy` → `cryptograPhy`